Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -194,4 +239,66 @@ extension PasswordCoordinator {
         }
         self.navigationController.pushViewController(viewController, animated: true)
     }
+    private func getPasswordFromKeychain(for identifier: Identifier) -> String? {
+        guard #available(iOS 11.3, *) else {
+            // Fallback to passsword login
+            return nil
+        }
+        var query = [String: Any]()
+        query[kSecClass as String] = kSecClassGenericPassword
+        query[kSecReturnData as String] = kCFBooleanTrue
+        query[kSecAttrAccount as String] = identifier.normalizedString as CFString
+        query[kSecAttrLabel as String] = "com.schibsted.account.biometrics.secrets" as CFString
+        query[kSecUseOperationPrompt as String] = "Please put your fingers on that button" as CFString
```

Comment:
Does this need localization?

Reply:
it seems like I never get a prompt here.

Reply:
This needs further tests on a real device.

Reply:
After a test on a real device it seems like I get a prompt there.